homestyle.css

```diff
@@ -0,0 +1,313 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+
+* {
+	box-sizing: border-box;
+}
+
+body { 
+	margin: 0; 
+	padding: 0;
+	font-family: 'Montserrat', sans-serif;
+	background-color: black;
+	color: #ffffff;
+}
+
+/* Navbar */
+
+.navbar {
+	width: auto;
+	max-width: 100%;
+	max-height: auto;
+	color: white;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 10px;
+	text-decoration: none;
+	letter-spacing: 3px;
+	font-size: 25px;
+}
+
+#NC {
+	color: white;
+	text-transform: uppercase;
+	padding: 15px;
+}
+
+.links {
+	display: flex;
+	margin: 10px;
+}
+
+.links a {
+  background-color: black;
+  border: #565656 solid 5px
+  color: white;
+  padding: 15px 50px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 20px;
+  text-transform: uppercase;
+  letter-spacing: 3px;
+  font-family: 'Montserrat', sans-serif;
+  color:white;
+}
+
+a:link{
+  text-decoration: none;
+}
+a:visited{
+  text-decoration: none;
+  color:white;
+}
+a:hover{
+  text-decoration: none;
+}
+a:focus{
+  text-decoration: none;
+}
+a:active{
+  text-decoration: none;
+}
+
+/* Home */
+
+.homebackground {
+	background: url(home.jpg) no-repeat;
+	width: 90%;
+	height: 70vh;
```

Comment:
I found a lot of problems when testing at different screen heights having used vh. How does this stand up on mobile browsers?